lib/intercept.js

```diff
@@ -123,6 +123,11 @@ function remove(interceptor) {
 }
 
 function removeAll() {
+  Object.keys(allInterceptors).forEach(function(key) {
+    allInterceptors[key].scopes.forEach(function(interceptor) {
```

Comment:
@disrupticons You're probably right that we have a weak guarantee that they're present. Feel free to skip this comment